src/components/HURUmap/Chart/renderChart.js

```diff
@@ -0,0 +1,26 @@
+import embed from "vega-embed";
+
+import configureDonutchart from "./donutScope";
+import configureBarchart from "./scope";
```

Comment:
Why do we call them "scope" and then rename `configureXXXX` on import? Asking for a friend.